dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -141,7 +141,7 @@ public Collection getCollectionByAlias(
     @UnitOfWork(readOnly = true)
     @Path("{organizationId}/collections/{collectionId}")
     @ApiOperation(value = "Retrieve a collection by ID.", notes = OPTIONAL_AUTH_MESSAGE, authorizations = {@Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Collection.class)
```

Comment:
I think `JWT_SECURITY_DEFINITION_NAME` is a constant for `BEARER` so you could use that instead to avoid declaring the new constant `OPENAPI_JWT_SECURITY_DEFINITION_NAME`